README.md

```diff
@@ -18,6 +18,8 @@ git clone https://github.com/fsr-itse/1327.git
 
 Freshly created code needs to be tested - besides our use of unit tests, linting and continous integration, it is possible to run the application in a non-production environment using *Vagrant* or a *Virtual Environment*.
 
+1327 runs on `Python 3.7` or higher.
```

Comment:
it runs on Python 3.6 with redis which will be there again after #764

Reply:
Aha!